### PR TITLE
Bump lib-core bindings to 0.5.1-rc.

### PR DESCRIPTION
**Pull request by phaazon**
Bump lib-core bindings to use 2.7.0-rc (bindings version 0.5.1-rc).

### Type

Feature.
Bug Fix: Android: avoid calling promises 2 times (https://github.com/LedgerHQ/djinni/pull/21)

### Context

None.

### Parts of the app affected / Test plan

I’m not sure. I make an Android build and tested on my phone, but not really deeply.

### CODE REVIEW

**Comment by gre** [comment]
@phaazon can you summarize up what this version is improving. it can be useful if we want to write a release notes in the future. thx

**Comment by gre** [comment]
if this is fixing bugs we should try to merge it for this sprint release & spend more time on QA, if it's only minor stuff, probably won't have the bandwidth to test it.

**Comment by phaazon** [comment]
@gre mostly ERC20 tokens, Ripple, password setting, memory optimization for dynamic objects (so those will consume less memory) and other features [listed here](https://github.com/LedgerHQ/lib-ledger-core/blob/develop/CHANGELOG.md) — I’ll complete it to add the few work for 2.7.0 that comprises a way to get the ERC20 tokens balance history, for instance.

**Comment by phaazon** [comment]
You can see the file [here](https://github.com/LedgerHQ/lib-ledger-core/blob/29549bc7204bb378754a0241d21f8096496b29a2/CHANGELOG.md)

**Comment by gre** [comment]
i'll test it a bit and merge.

**Comment by gre** [comment]
(holding this because crashes on my phone)

**Comment by phaazon** [comment]
Crashes mostly due to a bug that is likely to be fixed by [this](https://github.com/LedgerHQ/djinni/pull/21), but I would like to reproduce to validate my fix is a correct pach.

**Comment by gre** [comment]
i'll do a new PR, we've merged tons of things in develop
